taichi/program/kernel_profiler.h

```diff
@@ -12,18 +12,43 @@
 
 TLANG_NAMESPACE_BEGIN
 
+enum class KernelProfilerMode : int {
+#define PER_MODE(x) x,
+#include "taichi/inc/kernel_profiler_mode.inc.h"
+
+#undef PER_MODE
+};
+
+std::string kernel_profiler_name(KernelProfilerMode mode);
+
+KernelProfilerMode kernel_profiler_from_name(const std::string &mode);
+
 struct KernelProfileRecord {
   std::string name;
   int counter;
   double min;
   double max;
   double total;
+  double ldg;
```

Comment:
I'd appreciate more descriptive names for ldg and stg :P

Reply:
my bad :sob:  commit 291dd51